apps/common/frontend/src/components/beta/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Everything you have in common here should be scoped to the sv app alone. We wont share it across any other apps